schema.graphql

```diff
@@ -37,6 +37,37 @@ type GraphQLQuery {
     id: ID
   ): Actor!
   actors(eicFunctions: [EicFunction!]): [Actor!]!
+  esettExchangeEvent(
+    """The id of the exchange document."""
+    documentId: String!
```

Comment:
Should the graphql changes be in this PR?

Reply:
Yes, file is autogenerated, so I have included it here.